lib/releaselib.sh

```diff
@@ -631,13 +631,27 @@ release::gcs::locally_stage_release_artifacts() {
       || return 1
     release::gcs::stage_and_hash $gcs_stage $gci_path/configure.sh extra/gce \
       || return 1
+
     # shutdown.sh was introduced starting from v1.11 to make Preemptible COS nodes
     # on GCP not reboot immediately when terminated. Avoid including it in the release
     # bundle if it is not found (for backwards compatibility).
     if [[ -f $gci_path/shutdown.sh ]]; then
       release::gcs::stage_and_hash $gcs_stage $gci_path/shutdown.sh extra/gce \
        || return 1
     fi
+
+    # Having the Windows startup scripts from the GCE cluster deploy hosted with
+    # the release is useful for GKE.
+    win1803_local_path=$gce_path/win1803
+    win1803_gcs_path=extra/gce/win1803
+    release::gcs::stage_and_hash $gcs_stage $win1803_local_path/configure.ps1 $win1803_gcs_path \
```

Comment:
you should guard this based on either
- whether `$win1803_local_path` exists
- the kubernetes version

since older kubernetes releases would otherwise fail here if these files don't exist.

Reply:
Done